module-services/service-desktop/WorkerDesktop.cpp

```diff
@@ -113,95 +115,126 @@ void WorkerDesktop::reset()
 
 bool WorkerDesktop::handleMessage(uint32_t queueID)
 {
+    auto result       = false;
     auto &queue       = queues[queueID];
     const auto &qname = queue->GetQueueName();
 
     if (qname == sdesktop::RECEIVE_QUEUE_BUFFER_NAME) {
-        std::string *receivedMsg = nullptr;
-        if (!queue->Dequeue(&receivedMsg, 0)) {
-            LOG_ERROR("handleMessage failed to receive from \"%s\"", sdesktop::RECEIVE_QUEUE_BUFFER_NAME);
-            return false;
-        }
+        result = handleReceiveQueueMessage(queue);
+    }
+    else if (qname == sdesktop::SEND_QUEUE_BUFFER_NAME) {
+        result = handleSendQueueMessage(queue);
+    }
+    else if (qname == SERVICE_QUEUE_NAME) {
+        result = handleServiceQueueMessage(queue);
+    }
+    else if (qname == sdesktop::IRQ_QUEUE_BUFFER_NAME) {
+        result = handleIrqQueueMessage(queue);
+    }
+    else {
+        LOG_INFO("handeMessage got message on an unhandled queue");
+    }
 
-        using namespace sdesktop::endpoints;
-        auto factory = EndpointFactory::create(securityModel.getEndpointSecurity());
-        auto handler = std::make_unique<MessageHandler>(ownerService, messageProcessedCallback, std::move(factory));
+    return result;
+}
 
-        parser.setMessageHandler(std::move(handler));
-        parser.processMessage(std::move(*receivedMsg));
+bool WorkerDesktop::handleReceiveQueueMessage(std::shared_ptr<sys::WorkerQueue> &queue)
+{
+    if (not initialized) {
+        return false;
+    }
+    std::string *receivedMsg = nullptr;
+    if (!queue->Dequeue(&receivedMsg, 0)) {
```

Comment:
Logical not operators are not consistent. I'm not sure which one is better and whether or not there is a guideline about this. I have no preference. But we should at least use them consistently in the same function or maybe the entire file.

Side note: sometimes people put space after `!`, i.e.:
```
if (! queue->Dequeue(&receivedMsg, 0))
```
so this would be yet another style.

Reply:
done